notebooks/post_DL3_analysis.ipynb

```diff
@@ -1,36 +1,58 @@
 {
```

Comment:
<p>In order to avoid this problem I suggest to keep the school directory just as input but not writing anything in it. And create a new analysis directory were all the analysis products are stored</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>